tests/unit/test_solvers/test_idaklu_solver.py

```diff
@@ -46,13 +53,148 @@ def test_ida_roberts_klu(self):
             true_solution = 0.1 * solution.t
             np.testing.assert_array_almost_equal(solution.y[0, :], true_solution)
 
-    @unittest.skipIf(not pybamm.have_jax(), "jax or jaxlib is not installed")
+    def test_model_events(self):
+        for form in ["python", "casadi", "jax"]:
+            if form == "jax" and not pybamm.have_jax():
+                continue
```

Comment:
shouldn't this line not be caught in coverage?

Reply:
when we measure the coverage it won't get to the continue, but its needed for the other test suites, for which jax might not be available.

Reply:
well, its in the test, I assume its not necessary to cover the actual test code :) Who watches the watchers and all that....